notebooks/text_classification/solutions/text_generation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<ul><li>I'd remove all the TODOs and reserve them for the exercise notebook.</li><li>Maybe add a brief sentence above this class in a markdown cell, explaining that 1) we need to derive from tf.keras.Model, 2) in the constructor we define the layers, and 3) in the call method we define the pass forward using the layers defined in the constructor</li><li>I'd remove line 14: x = inputs, and pass directly inputs to self.embedding</li><li>Maybe add a comment on over line 16, explaining what that state logic in 16-18 does (here I think it's warranted to add the comment in the code directly). Maybe this can also be explained in a markdown cell.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Done! Took all your suggestions!</p>